[2024-12-05T18:37:49.489+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-05T18:37:49.499+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_recursion.get_vars manual__2024-12-05T18:37:47.687764+00:00 [queued]>
[2024-12-05T18:37:49.504+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_recursion.get_vars manual__2024-12-05T18:37:47.687764+00:00 [queued]>
[2024-12-05T18:37:49.504+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-05T18:37:49.514+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): get_vars> on 2024-12-05 18:37:47.687764+00:00
[2024-12-05T18:37:49.519+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=132) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-05T18:37:49.521+0000] {standard_task_runner.py:72} INFO - Started process 134 to run task
[2024-12-05T18:37:49.521+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'test_recursion', 'get_vars', 'manual__2024-12-05T18:37:47.687764+00:00', '--job-id', '384', '--raw', '--subdir', 'DAGS_FOLDER/test_recursion.py', '--cfg-path', '/tmp/tmp0a7t_4o2']
[2024-12-05T18:37:49.522+0000] {standard_task_runner.py:105} INFO - Job 384: Subtask get_vars
[2024-12-05T18:37:49.552+0000] {task_command.py:467} INFO - Running <TaskInstance: test_recursion.get_vars manual__2024-12-05T18:37:47.687764+00:00 [running]> on host c7780884450d
[2024-12-05T18:37:49.719+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='artzsh' AIRFLOW_CTX_DAG_ID='test_recursion' AIRFLOW_CTX_TASK_ID='get_vars' AIRFLOW_CTX_EXECUTION_DATE='2024-12-05T18:37:47.687764+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-05T18:37:47.687764+00:00'
[2024-12-05T18:37:49.720+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-05T18:37:49.751+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-05T18:37:49.756+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-05T18:37:49.757+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=test_recursion, task_id=get_vars, run_id=manual__2024-12-05T18:37:47.687764+00:00, execution_date=20241205T183747, start_date=20241205T183749, end_date=20241205T183749
[2024-12-05T18:37:49.775+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-05T18:37:49.787+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-05T18:37:49.790+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
